src/components/calculator.js

```diff
@@ -1,45 +1,47 @@
 import React from 'react';
 import calculate from '../logic/calculate';
+import { useState } from 'react';
 
-class Calculator extends React.Component {
-    constructor(props) {
-        super(props)
-        this.state = {};
-        this.handleEvent = this.handleEvent.bind(this); 
-    }
 
-    handleEvent(e) {
-        this.setState((state) => calculate(state, e.target.textContent));
-    }
-  render() {
-      const {total, process, next} = this.state;
+
+    const Calculator = () => {
```

Comment:
- [ ] [REQUIRED] Great refactor @wadaudzai , however, you need to work on your display as it does not show the operators on the screen.
- [ ] [OPTIONAL] Also, ensure that the calculator is placed at the center of the page for a better UI.

### Take a look at this

[Video](https://vimeo.com/748167871/60125ab9c0)